### DISCUSSION TITLE
请问dumpbin not found如何解决

### DISCUSSION THREAD

**Discussion by jz315** [xmake-io/xmake#4380] [Q&A] [answered]
`error: ...mdir\core\sandbox\modules\import\core\base\scheduler.lua:56: dumpbin not found!
stack traceback:
        @programdir\core\base\utils.lua:290: in function <@programdir\core\base\utils.lua:280>
        [C]: in function 'error'
        @programdir\core\base\os.lua:923: in function 'base/os.raiselevel'
        (...tail calls...)
        ...mdir\core\sandbox\modules\import\core\base\scheduler.lua:56: in field 'co_start_withopt'
        @programdir\modules\async\runjobs.lua:217: in function <@programdir\modules\async\runjobs.lua:162>
        [C]: in function 'xpcall'
        @programdir\core\base\utils.lua:280: in function 'base/utils.trycall'
        @programdir\core\base\scheduler.lua:520: in function 'base/scheduler.co_group_begin'
        ...mdir\core\sandbox\modules\import\core\base\scheduler.lua:94: in field 'co_group_begin'
        @programdir\modules\async\runjobs.lua:162: in function <@programdir\modules\async\runjobs.lua:56>
        (...tail calls...)
        ...     (skipping 1 levels)
        @programdir\actions\build\main.lua:148: in function <@programdir\actions\build\main.lua:142>
        [C]: in function 'xpcall'
        @programdir\core\base\utils.lua:280: in function 'base/utils.trycall'
        @programdir\core\sandbox\modules\try.lua:117: in global 'try'
        @programdir\actions\build\main.lua:140: in global 'build_targets'
        @programdir\actions\build\main.lua:224: in function <@programdir\actions\build\main.lua:188>
        (...tail calls...)
        [C]: in function 'xpcall'
        @programdir\core\base\utils.lua:280: in function 'base/utils.trycall'
        (...tail calls...)
        @programdir\core\base\task.lua:501: in function 'base/task.run'
        @programdir\core\main.lua:307: in upvalue 'cotask'
        @programdir\core\base\scheduler.lua:404: in function <@programdir\core\base\scheduler.lua:397>`

已经添加了环境变量，在powershell里可以正常使用dumpbin

**Comment by waruqi** [comment]
给下完整 -vD logs 和 xmake.lua

**Reply by jz315** [reply, marked as answer]
重启电脑后解决了
